From fc9a0b36d8627afc244860b0b7df109e95092a54 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Fri, 12 Sep 2014 11:48:42 -0400
Subject: [ethernet] i40e: Remove unnecessary assignment

Message-id: <1410522528-19793-5-git-send-email-sassmann@redhat.com>
Patchwork-id: 93937
O-Subject: [RHEL7.1 PATCH 04/10] i40e: Remove unnecessary assignment
Bugzilla: 1140994
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>

From: Serey Kong <serey.kong@intel.com>

Remove unnecessary setting of "ret" variable as it's already set at
the top of the function.

Change-ID: Icaccfc67f335817a23579b7c43625d59ad6c9925
Signed-off-by: Serey Kong <serey.kong@intel.com>
Tested-by: Jim Young <jamesx.m.young@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 356821a37dee93317779e39c781a5c103565c508)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
index 7be54ae..1bf6e2e 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
@@ -2098,7 +2098,6 @@ int i40e_ndo_set_vf_mac(struct net_device *netdev, int vf_id, u8 *mac)
  /* Force the VF driver stop so it has to reload with new MAC address */
  i40e_vc_disable_vf(pf, vf);
  dev_info(&pf->pdev->dev, "Reload the VF driver to make this change effective.\n");
- ret = 0;
 
 error_param:
  return ret;
-- 
1.7.1